meta-resin-common/classes/resin-sanity.bbclass

```diff
@@ -10,7 +10,9 @@ def resinos_build_configuration():
 	success = True
 	if d.getVar('PACKAGE_CLASSES', True) != "package_ipk":
 		bb.warn("ResinOS distro depends on opkg packages (ipk). Make sure PACKAGE_CLASSES is set on package_ipk.")
-	if d.getVar('DOCKER_STORAGE', True) not in ['aufs', 'overlay2']:
+	if d.getVar('DOCKER_STORAGE', True):
+		bb.warn("DOCKER_STORAGE variable was replaced by BALENA_STORAGE. Please update your build configuration.")
```

Comment:
Why is it warn, and not an error?
Is it because the change in `resin-os.inc` below converts one to the other?

Reply:
Yes. The build system for now throws a warning and handles the case where the old variables is still used (see resin-os.inc). In the future this will be replaced by an error.